src/main/java/io/cryostat/recordings/RecordingHelper.java

```diff
@@ -557,6 +561,7 @@ public List<ArchivedRecording> listArchivedRecordings() {
                                     getArchivedRecordingMetadata(jvmId, filename)
                                             .orElseGet(Metadata::empty);
                             return new ArchivedRecording(
+                                    jvmId,
```

Comment:
Ah, here is the structure corresponding to this, right? https://github.com/cryostatio/cryostat-web/pull/1227#discussion_r1550293676